integration-tests/schemas/createSchema.test.js

```diff
@@ -0,0 +1,38 @@
+const assert = require("assert");
+
+const ApiWrapper = require("./scaffolding/api/apiWrapper");
+const { getFrequencyAPI, getSignerAccountKeys} = require("./scaffolding/api/apiConnection");
+
+const { AVRO_GRAPH_CHANGE } = require("./scaffolding/fixtures/schemaTypes");
+const { AVRO, PARQUET } = require("./scaffolding/fixtures/modelTypes");
+const { ON_CHAIN, IPFS } = require("./scaffolding/fixtures/payloadLocation");
+
+
+describe("#createSchema", () => {
+    let api;
+
+    beforeEach(async () => {
+        api = new ApiWrapper(await getFrequencyAPI(), getSignerAccountKeys());
+    });
+
+    it("should successfully create an Avro GraphChange schema", async () => {
+        await api.createSchema(AVRO_GRAPH_CHANGE, AVRO, ON_CHAIN);
```

Comment:
how are we going to manage state?
Are we running integrations against a long lived chain instance or we spin up a new chain every time we are running integration test? There are pros and cons for each approach.

 If we are running against a long lived instance then the input should be generated randomly so you don't have conflict and error after  the first run.

Reply:
these tests run against a locally running chain. we could boot up a chain instance for each test but that would increase the time needed for the suite to run.

for now, iv gotten around the conflicts you mention by incrementing the nonce value.